### PR TITLE
TH-1065: Adds audience age info to single course page

### PR DESCRIPTION
**Pull request by melniiv**
## Additional notes :spiral_notepad:

### CODE REVIEW

File: src/domain/event/eventInfo/__tests__/EventInfo.test.tsx [hunk 1]
```diff
@@ -206,3 +210,45 @@ it('should create ics file succesfully when end time is not defined', async () =
 
   expect(FileSaver.saveAs).toBeCalled();
 });
+
+it('should show audience age info in signle course page', async () => {
```
Comment (nenonja) [inline src/domain/event/eventInfo/__tests__/EventInfo.test.tsx:214]: Very well coveraged tests! 👍

Comment (nenonja) [reply src/domain/event/eventInfo/__tests__/EventInfo.test.tsx:214]: Perhaps one case can be added for the test:
`return ageLimit !== '0+' ? `${ageLimit} -${t('event.info.age')}` : '';`
should not render the text. Perhaps checking that age title is not present.

**Review by nenonja — COMMENTED** [review]
Looks good! One test case could be added.

File: src/domain/event/eventInfo/__tests__/EventInfo.test.tsx (hunk 1 above)
Comment (melniiv) [reply src/domain/event/eventInfo/__tests__/EventInfo.test.tsx:214]: true that. i ll add

Comment (melniiv) [reply src/domain/event/eventInfo/__tests__/EventInfo.test.tsx]: fixed

File: src/domain/event/eventInfo/__tests__/EventInfo.test.tsx [hunk 2]
```diff
@@ -206,3 +210,60 @@ it('should create ics file succesfully when end time is not defined', async () =
...
+    <EventInfo
+      event={{ ...event, audienceMinAge: null, audienceMaxAge: null }}
+      eventType="course"
+    />,
+    {
+      routes: [`/fi/courses`],
+    }
+  );
+  await actWait();
+
```
Comment (nenonja) [inline src/domain/event/eventInfo/__tests__/EventInfo.test.tsx:258]: There could also check that the title is not present: 
expect(screen.queryByText(/Ikäryhmä/i)).not.toBeInTheDocument();

Comment (melniiv) [reply src/domain/event/eventInfo/__tests__/EventInfo.test.tsx:258]: done

**Review by nenonja — APPROVED** [review]
Approved, just thinking that should we hold the merging the pr now as we should first merge events-helsinki-2 to master, otherwise all the conflicts are left to this mega pr: https://github.com/City-of-Helsinki/events-helsinki-ui/pull/249
I was hoping to merging it today....

**Pull request merged**